docs/nodes/maintain/avalanchego-config-flags.md

```diff
@@ -602,10 +602,14 @@ Weight to provide to each peer when staking is disabled. Defaults to `1`.
 ## Subnets
 
 ### Whitelist
-
 #### `--whitelisted-subnets` (string)
+Deprecated
```

Comment:
See example how to mark deprecation: https://docs.avax.network/apis/avalanchego/apis/c-chain#avaximportavax